packages/SwingSet/bin/extract-transcript-from-kerneldb.js

```diff
@@ -0,0 +1,109 @@
+import lmdb from 'node-lmdb';
+import process from 'process';
+import fs from 'fs';
+
+const argv = process.argv.splice(2);
+const dirPath = argv[0];
+const vatName = argv[1];
+
+if (!dirPath) {
+  console.log('extract-transcript-from-kerneldb DBDIR : list all vats in DB');
+  console.log(
+    'extract-transcript-from-kerneldb DBDIR VATID : extract transcript',
+  );
+  process.exit(0);
+}
+
+const lmdbEnv = new lmdb.Env();
+lmdbEnv.open({
+  path: dirPath,
+  mapSize: 2 * 1024 * 1024 * 1024, // XXX need to tune this
+});
+
+const dbi = lmdbEnv.openDbi({
+  name: 'swingset-kernel-state',
+  create: false,
+});
+const txn = lmdbEnv.beginTxn();
+function get(key) {
+  return txn.getString(dbi, key);
+}
```

Comment:
Seems to me this should use the LMDB swingstore rather than the raw LMDB API.

Reply:
Yeah, I had an alternate version which used that, but `swing-store-lmdb` uses `assert`, which requires some form of `@agoric/install-ses`, which was interfering with my attempts to debug the tool. I'll investigate the consequences of moving it to `swing-store-lmdb` the next time I work on it.